### PR TITLE
Corrected 3 lines for Readability in response.py

### PR DESCRIPTION
**Pull request by shallow-source**
Merging of multiple isinstance calls to isinstance(self.data(), (dict, list)) 
Issue #866

### CODE REVIEW

File: masonite/response.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ def converted_data(self):
         Returns:
             string -- Returns a string representation of the data
         """
-        if isinstance(self.data(), dict) or isinstance(self.data(), list):
+        if isinstance(self.data(), (dict,list)):
```
Comment (vaibhavmule) [inline masonite/response.py]: Can you just add space after the comma
```suggestion
        if isinstance(self.data(), (dict, list)):
```

File: masonite/response.py [hunk 2]
```diff
@@ -89,9 +89,9 @@ def view(self, view, status=200):
         if not self.request.get_status():
             self.request.status(status)
 
-        if isinstance(view, dict) or isinstance(view, list):
+        if isinstance(view, (dict,list)):
```
Comment (vaibhavmule) [inline masonite/response.py]: add space after comma

File: masonite/response.py [hunk 3]
```diff
@@ -89,9 +89,9 @@ def view(self, view, status=200):
...
             return self.json(view, status=self.request.get_status())
-        elif isinstance(view, Collection) or isinstance(view, Model):
+        elif isinstance(view, (Collection,Model)):
```
Comment (vaibhavmule) [inline masonite/response.py]: same, add space after a comma

File: masonite/response.py (hunk 1 above)
Comment (shallow-source) [reply masonite/response.py]: sure.

Comment (shallow-source) [reply masonite/response.py]: done

File: masonite/response.py (hunk 2 above)
Comment (shallow-source) [reply masonite/response.py]: done

File: masonite/response.py (hunk 3 above)
Comment (shallow-source) [reply masonite/response.py]: done

**Pull request merged**